### PR TITLE
Test coverage increase

### PR DESCRIPTION
**Pull request by dnlup**
Increase a little coverage of unit tests.

Ref: #2486 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] ~tests and/or benchmarks are included~
- [ ] ~documentation is changed or added~
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: lib/handleRequest.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function handler (request, reply) {
 function preValidationCallback (err, request, reply) {
   if (reply.sent === true ||
     reply.raw.writableEnded === true ||
-    (reply.raw.writable === false && reply.raw.finished === true)) return
```
Comment (dnlup) [inline lib/handleRequest.js:80]: `reply.raw.finished` never gets covered, this makes me think that checking `writable` should be enough.

This is something I added in https://github.com/fastify/fastify/pull/2233

File: lib/handleRequest.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ function preValidationCallback (err, request, reply) {
 function preHandlerCallback (err, request, reply) {
   if (reply.sent ||
     reply.raw.writableEnded === true ||
-    (reply.raw.writable === false && reply.raw.finished === true)) return
```
Comment (dnlup) [inline lib/handleRequest.js:114]: Same as https://github.com/fastify/fastify/pull/2542/files#r484694171

File: lib/fourOhFour.js [hunk 3]
```diff
@@ -147,7 +147,7 @@ function fourOhFour (options) {
     this[kFourOhFourLevelInstance][kFourOhFourContext] = context
 
     router.all(prefix + (prefix.endsWith('/') ? '*' : '/*'), routeHandler, context)
-    router.all(prefix || '/', routeHandler, context)
```
Comment (dnlup) [inline lib/fourOhFour.js:150]: The slash is assigned to `prefix` as a default a few lines before already, so this branch is never taken.

**Review by L2jLiga — APPROVED** [review]
Lgtm

**Review by mcollina — APPROVED** [review]
lgtm

**Review by fox1t — APPROVED** [review]
LGTM

**Pull request merged**